### PR TITLE
Klyqa

### PR DESCRIPTION
**Pull request by fraizy22**
## Proposed change

Documentation for the new klyqa integration.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/78601
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/3664
- This PR fixes or closes issue: fixes #

## Checklist

- [x ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/klyqa.markdown [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+ha_release: 2022.12
+ha_iot_class: Local Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@fraizy22'
+ha_domain: klyqa
+ha_homekit: true
+ha_platforms:
+  - light
+  - vacuum
```
Comment (frenck) [inline source/_integrations/klyqa.markdown]: Initial PRs to Home Assistant can only contain 1 platform at a time (this applies both to the documentation and the Core PRs).

File: source/_integrations/klyqa.markdown [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+ha_integration_type: integration
+---
+
+The `klyqa` integration will allow users to integrate with their Klyqa Account their Klyqa smart devices into Home Assistant and use it to automate their homes.
```
Comment (frenck) [inline source/_integrations/klyqa.markdown]:
```suggestion
The Klyqa integration will allow users to integrate with their Klyqa account with their Klyqa smart devices.
```

Remove the latter bit as well, as the is kinda assumed to be the case :)

**Review by frenck — CHANGES_REQUESTED** [review]
Set up instructions are missing, did you forget to include the `config_flow.md` template?

../Frenck

**Comment by fraizy22** [comment]
> Set up instructions are missing, did you forget to include the `config_flow.md` template?
> 
> ../Frenck

Ok I added the line. Was this the right one?

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed.

**Pull request closed without merging**